lib/awesome_print/core_ext/kernel.rb

```diff
@@ -1,8 +1,3 @@
-# Copyright (c) 2010-2013 Michael Dvorkin
```

Comment:
:+1:  getting rid of the superfluous copyright lines. It's not necessary given a master license.